sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -1161,7 +1160,7 @@ private case class GpuOrcFileFilterHandler(
     val conf = broadcastedConf.value.value
     OrcConf.IS_SCHEMA_EVOLUTION_CASE_SENSITIVE.setBoolean(conf, isCaseSensitive)
 
-    val filePath = new Path(new URI(partFile.filePath))
+    val filePath = GpuSparkPath(partFile.filePath.toString).toPath
```

Comment:
Why create a GpuSparkPath when we're already forcing the file path to be a string?  Seems like this could be simplified to:
```suggestion
    val filePath = new Path(new URI(partFile.filePath.toString))
```

Comment applies to the other places this code pattern shows up.